src/main/java/edu/iis/powp/adapter/MyAdapter.java

```diff
@@ -8,9 +8,9 @@
 /**
  * Plotter adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements IPlotter {
+public class MyAdapter  implements IPlotter {
 	private int startX = 0, startY = 0;
-
+	DrawPanelController locDrawer = DrawerFeature.getDrawerController();

```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.

The attribute should be private.